Pipfile

```diff
@@ -77,6 +77,7 @@ markupsafe = "*"
 pdfkit = "*"
 ordered-set = "*"
 cachetools = "*"
+types-simplejson = "*"
```

Comment:
When adding this package, it did flag some additional type hinting errors in different folders which may help us have better type hints. For example with the package it shows this error:

<img width="1277" alt="Screenshot 2023-11-29 at 15 55 00" src="https://github.com/ONSdigital/eq-questionnaire-runner/assets/144226112/a3ab04e7-c3e9-47a8-aa6c-fbb52caa8ddb">

But without the package, it doesn't. Would this be considered valuable? Looking at the type hinting already in `feedback.py` and what it's suggesting may not be valuable, what are your thoughts?

Reply:
Those extra ones are not because of that package. These extra are because mypy was updated when you added the new package. If you pin mypy, these errors should go away. They are fixed in https://github.com/ONSdigital/eq-questionnaire-runner/pull/1269